package.json

```diff
@@ -21,10 +21,11 @@
     "@fortawesome/free-brands-svg-icons": "^5.15.2",
     "@fortawesome/free-solid-svg-icons": "^5.3.1",
     "@fortawesome/react-fontawesome": "^0.1.3",
-    "@material-ui/core": "^4.11.3",
+    "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.57",
     "@rjsf/core": "^3.0.0",
+    "@sage-bionetworks/react-base-table": "^1.12.3",
```

Comment:
https://github.com/Sage-Bionetworks/react-base-table is forked from https://github.com/Autodesk/react-base-table